app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AppointAgent.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.voapropertylinking.models.modernised.agentrepresentation
+
+import play.api.libs.json.{Json, OFormat}
+
+case class AppointAgent(agentRepresentativeCode: Long, scope: String) {}
```

Comment:
this object is missing the propertyLinkSubmissionId's.

Reply:
I have not implemented that. I was told that we would be joining the old journey for that scenario. I have not catered for "choose from list"